ports/servoshell/app.rs

```diff
@@ -112,10 +112,6 @@ impl App {
         // frame, set this to false, so we can avoid an unnecessary recomposite.
         let mut need_recomposite = true;
 
-        // If we have a minibrowser, ask the compositor to notify us when a new frame
-        // is ready to present, so that we can paint the minibrowser then present.
```

Comment:
this explanation is important for understanding the interaction between servo and the minibrowser when drawing and presenting frames. can we update it, instead of removing it?

Reply:
I tried to move this explanation to doc comment in all variants of `PumpResult`. wdyt?

Reply:
looks good!